canvas_account_admin_tools/settings/base.py

```diff
@@ -499,6 +508,11 @@
     'masquerade_session_minutes': SECURE_SETTINGS.get('masquerade_session_minutes', 60),
 }
 
+SELF_ENROLLMENT_TOOL_RSETTINGS = {
```

Comment:
Typo
Are these settings being referenced in the code?

Reply:
Good catch. This was originally sued for rendering the published enrollment url. But since Patrick will handle it in the other story, I will clean it up here.